packages/driver/cypress/integration/commands/actions/check_spec.js

```diff
@@ -2,18 +2,8 @@ const { assertLogLength } = require('../../../support/utils')
 const { _, Promise, $ } = Cypress
 
 describe('src/cy/commands/actions/check', () => {
-  before(() => {
-    cy
-    .visit('/fixtures/dom.html')
-    .then(function (win) {
-      this.body = win.document.body.outerHTML
-    })
-  })
-
   beforeEach(function () {
-    const doc = cy.state('document')
-
-    $(doc.body).empty().html(this.body)
```

Comment:
There are many identical changes like this, I see they came from this commit https://github.com/cypress-io/cypress/commit/dcbc68b8e7a4d5cab186694da3781ac63a44db77 - @mschile can you shed some light on why these changes became possible? Is this just a cleanup of unused code, or did you move it somewhere?

Reply:
As discussed below, all of these changes were a result of turning on session support which visits `about:blank` between` tests.